src/translate-adapter-handlers.ts

```diff
@@ -194,16 +204,95 @@ async function translateIoPackage(): Promise<void> {
 	if (content.common.messages) {
 		console.log("Translate Messages");
 		for (const message of content.common.messages) {
-			console.log(`   Message: ${message.title.en}`);
+			console.log(gray(`   Message: ${message.title.en}`));
 			await translateNotExisting(message.title);
 			await translateNotExisting(message.text);
-			await translateNotExisting(message.linkText);
+			// test first if there is a linkText - since it's not mandatory
+			if (message.linkText) await translateNotExisting(message.linkText);
 		}
 	}
 	await writeJson(ioPackage, content, { spaces: 4, EOL });
 	console.log(`Successfully updated ${path.relative(".", ioPackage)}`);
 }
 
+async function loopJSON(content: object): Promise<object> {
+	const baseContent: object[] = [];
+	// read all existing language files
+	for (let n = 0; n < i18nBases.length; n++) {
+		if (existsSync(i18nBases[n])) {
+			baseContent.push(await readJson(i18nBases[n]));
```

Comment:
Is this the right place to read the json? loopJson method is called recursie ... so we always read that. Should it not be done once and maybe passed to loopJson as parameter?